QA.md

```diff
@@ -123,3 +123,268 @@
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TTT - Test Token Test` name should be in gray
+        1. The `TN1 - Test Nft` name should be in gray
+        1. The `TNT - Test Nft Test` name should be in gray
+    1. Click on the `TTT - Test Token Test` item
+        1. Nothing should happen
+    1. Click on the `TN1 - Test Nft` item
+        1. Nothing should happen
+    1. Click on the `TNT - Test Nft Test` item
+        1. Nothing should happen
+    1. Click on the `HTR - HATHOR` item
+        1. The **Balance** page should open
+1. **Register `TTT` token and send 2 tokens after turn on `Show amounts on notification` option**
+    > WARNING: Not possible using `wallet-service` due to a validation that consults the fullnode.
+    > Jump to **Test open the wallet 2 weeks later** (19)
+    1. Register `TTT` token in the wallet
+    1. Turn on the `push-notification` feature toggle
+    1. Go to the **Push Notification** page
+    1. Turn **on** the `Enable Push Notification` option
+    1. Turn **on** the `Show amounts on notification` option
+    1. Send HTR and TTT to this wallet
+        1. Wait 3-5 min
+        1. You should receive a notification of new transaction showing 2 amounts in the message
+            > You have received 0.02 TTT and 0.01 HTR on a new transaction.
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TTT - Test Token Test` name should be in the primary color (purple)
+    1. Click on the `TTT - Test Token Test` item
+        1. The **Balance** page for `TTT` token should open
+1. **Register `TN1` token and send 2 tokens after turn on `Show amounts on notification` option**
+    1. Register `TTT` token in the wallet
+    1. Turn on the `push-notification` feature toggle
+    1. Go to the **Push Notification** page
+    1. Turn **on** the `Enable Push Notification` option
+    1. Turn **on** the `Show amounts on notification` option
+    1. Send HTR and TN1 to this wallet
+        1. Wait 3-5 min
+        1. You should receive a notification of new transaction showing 2 amounts in the message
+            > You have received 0.03 TN1 and 0.02 HTR on a new transaction.
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TN1 - Test Nft` name should be in the primary color (purple)
+        1. The `TN1 - Test Nft` amount should be integer
+    1. Click on the `TN1 - Test Nft` item
+        1. The **Balance** page for `TN1` token should open
+1. **Send token to self**
+    1. Wait 3-5 min
+    1. You should **not** receive a notification of new transaction
+1. Test open the wallet 2 weeks later
```

Comment:
We usually run the QA after sending to the stores for beta versions. You should mark this item here to be done only if it's running locally

Reply:
[6638db6](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/6638db6d32dc3a51876cbd3c8b180181fd321dbc)